Tests/UITests/TestApp/Views/OnboardingWelcomeTestView.swift

```diff
@@ -36,14 +36,3 @@ struct OnboardingWelcomeTestView: View {
             #endif
     }
 }
-
-
-#if DEBUG
-#Preview {
-    OnboardingStack(startAtStep: OnboardingWelcomeTestView.self) {
-        for onboardingView in OnboardingFlow.previewSimulatorViews {
```

Comment:
If we remove all of these previews, is `OnboardingFlow.previewSimulatorViews` still required?